_plugins/end-of-life-filters.rb

```diff
@@ -56,6 +56,46 @@ def remove_first_element(html, element)
   def drop_zero_patch(input)
     input.end_with?(".0") ? input[0, input.length - 2] : input
   end
+
+  # Compute the number of days from now to the given date.
+  #
+  # Usage (assuming now is '2023-01-01'):
+  # {{ '2023-01-10' | days_from_now }} => 9
+  # {{ '2023-01-01' | days_from_now }} => 0
+  # {{ '2022-12-31' | days_from_now }} => -1
+  def days_from_now(from)
+    from_timestamp = Date.parse(from.to_s).to_time.to_i
+    to_timestamp = Date.today.to_time.to_i
+    return (from_timestamp - to_timestamp) / (60 * 60 * 24)
+  end
+
+  # Compute the color according to the given number of days until the end.
+  #
+  # Usage:
+  # {{ true | end_color }} => bg-green-000
+  # {{ false | end_color }} => bg-red-000
+  # {{ '2025-01-01' | days_until_end: '2023-01-01' | end_color }} => bg-green-000
```

Comment:
`days_until_end` is missing?

Reply:
Forgot to update the doc: dates are now directly supported by the filter.

Reply:
Fixed